File: Easy/LCE - 1588.js
/**
 * @param {number[]} arr
 * @return {number}
 */
var sumOddLengthSubarrays = function(arr) {
    
    let ans = 0
    
    for(let i=1; i<=arr.length; i++) {
        for(let j=0; j<arr.length-i+1; j++) {
            let temp = arr.slice(j,j+i)
            if(temp.length%2==1){
                ans += sum(temp)
            }
        }
    }
    
    return ans
    
    function sum(arr) {
        let ans = arr.reduce((a,b) => a+b)
        return ans
    }  
};

File: Easy/Python - Solutions/LCE_1588.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        ans = 0
        k = 1

        while k <= len(arr):
            for i in range(0, len(arr)):
                if i+k <= len(arr):
                    ans += sum(i for i in arr[i:i+k])

            k += 2

        return ans
